contrib/systemd/containerized-installs/openvswitch.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+After=docker.service
+Requires=docker.service
+Before=origin-master.service
+
+[Service]
+EnvironmentFile=/etc/sysconfig/origin-node
+ExecStartPre=-/usr/bin/docker rm -f openvswitch
+ExecStart=/usr/bin/docker run --name openvswitch --rm --privileged --net=host --pid=host -v /lib/modules:/lib/modules -v /run:/run -v /sys:/sys:ro -v /etc/origin/openvswitch:/etc/openvswitch:Z openshift/openvswitch
```

Comment:
The :Z scares me, given --privileged, is it needed? This is going to relabel files in /etc....